### ISSUE TITLE
`fable-compiler-js` only includes the `fable-library-js`

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/Fable#3827]
### Description

There an inconsistency in `fable-compiler-js`.

Indeed, that CLI tool can compile to any output target however, it only includes the `fable-libary` for the JavaScript version of Fable. 

We should either includes the other `fable-library` versions or allows that tool to only support JavaScript like target (JavaScript and TypeScript). If we choose the later it mean we still needs to also include `fable-library-ts` in the distributed package.

**Comment by ncave** [comment]
@MangelMaxime 
> We should either includes the other fable-library

Yes, we should be including all fable libraries, and copying them to locally to output folder on compile, same as the .NET Fable CLI does.
